qutebrowser/mainwindow/mainwindow.py

```diff
@@ -145,6 +145,17 @@ class MainWindow(QWidget):
         _private: Whether the window is in private browsing mode.
     """
 
+    # Application wide stylesheets
+    STYLESHEET = """
+        QLabel#hint {
```

Comment:
Would using `HintLabel` here instead (without `setObjectName`) work?

Reply:
Huh, it seems to work, that's definitely cleaner.